Tests/LibMarkdown/TestImageSizeExtension.cpp

```diff
@@ -15,17 +15,17 @@ struct TestCase {
 
 static constexpr Array image_size_tests {
     // No image size:
-    TestCase { .markdown = "![](foo.png)"sv, .expected_html = R"(<p><img src="foo.png" alt="" ></p>)"sv },
+    TestCase { .markdown = "![](foo.png)"sv, .expected_html = R"(<p><img src="foo.png" alt="" /></p>)"sv },
     // Only width given:
-    TestCase { .markdown = "![](foo.png =100x)"sv, .expected_html = R"(<p><img src="foo.png" style="width: 100px;" alt="" ></p>)"sv },
+    TestCase { .markdown = "![](foo.png =100x)"sv, .expected_html = R"(<p><img src="foo.png" style="width: 100px;" alt="" /></p>)"sv },
     // Only height given:
-    TestCase { .markdown = "![](foo.png =x200)"sv, .expected_html = R"(<p><img src="foo.png" style="height: 200px;" alt="" ></p>)"sv },
+    TestCase { .markdown = "![](foo.png =x200)"sv, .expected_html = R"(<p><img src="foo.png" style="height: 200px;" alt="" /></p>)"sv },
     // Both width and height given
-    TestCase { .markdown = "![](foo.png =50x25)"sv, .expected_html = R"(<p><img src="foo.png" style="width: 50px;height: 25px;" alt="" ></p>)"sv },
+    TestCase { .markdown = "![](foo.png =50x25)"sv, .expected_html = R"(<p><img src="foo.png" style="width: 50px;height: 25px;" alt="" /></p>)"sv },
```

Comment:
:raised_eyebrow: Self-closing tags in HTML are not self-closing because they have `<tag />`. They are self-closing because they are in the list of self-closing tags. the `/` is explicitly ignored in HTML5. If this has an actual impact on the rendered HTML then that's a bug in the CommonMark test suite.

see https://jakearchibald.com/2023/against-self-closing-tags-in-html/